k8s/demo/common/bsp/bsp-sample-app.yaml

```diff
@@ -36,7 +36,7 @@ spec:
       bsoIdamRedirectUrl: "https://bulk-scan-orchestrator-demo.service.core-compute-demo.internal/oauth2/callback"
       devMode: true
     java:
-      image: hmctspublic.azurecr.io/bulk-scan/sample-app:prod-1a9095f3
+      image: hmctspublic.azurecr.io/bulk-scan/sample-app:prod-1a9095f3   #{"$imagepolicy": "flux-system:bsp-sample-app"}
```

Comment:
Can this be reviewed? Using older image & if we apply image policy - it may be updated

Reply:
bsp-sample-app now removed as product chart no longer required.